week2/day4_week2.Rmd

```diff
@@ -0,0 +1,93 @@
+---
+title: "Day 4 Week 2 assignment"
+author: "Alexandra Roffe"
+date: 2022-06-09
+output: html_document
+---
+
+day 4, week 2
+next textbook chapter 2. assignment 2.2 and 2.5
+
+2.2: Heart transplants, Past II
+a) What proportion of the patients in the treatment group and the control group died?
+```{r}
+# in the treatment group 
+45/69
+30/34
+```
+
+b) 
+i. The claims being tested are:
+  
+  null: the treatment has no effect on whether the patient dies or not. The difference in the proportions of the treated group and the control group is due to chance. 
+  
+  alternative: The treatment has an effect on whether the patient dies or not. The difference in the proportions of the treated group and the control group is due to the treatment. 
+  
+ii. 28, 75, 69, 34, 75/103 = 0.728,  30/34 - 45/69 = 0.230179, 
+
+iii. 
+  mean of chart is 0.01. These simulations show us that the treatment is effective. The fraction (difference in the proportions) is low, so we reject the null hypothesis in favor of the alternative. 
+  
+  
+2.5) Social experiment 
+
+a) 
+  null would same distribution, average is 0. The p(pr_sim) = p(con_sim). 
+
+  alternative is that the average is not equal to zero; the p(pr_sim) != p(con_sim).
+  
+b) 
+```{r}
+5/20 - 15/25
+```
+c)
+The graph shows us the probability of getting -0.35 is around 0.025 (2.5%). 
+This is less than alpha so reject the null hypothesis. 
+```{r}
+prop.test(3500, 10000) ## ???
+```
+the p-value < 2.2e-16. 
+We are not rejecting the null
+

```

Comment:
prop.test() takes a vector of successes and failures, not really sure what you're doing here since you answered the question above.